playbooks/ansible-test-integration-base/pre.yaml

```diff
@@ -29,24 +29,3 @@
     - name: Install ansible into virtualenv
       shell: "~/venv/bin/pip install {{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}"
       when: ansible_test_collections is not defined
-
-    - name: Enable ARA callback plugin
-      ini_file:
-        path: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}/test/integration/{{ ansible_test_command }}.cfg"
-        section: defaults
-        option: callback_plugins
-        value: "{{ ansible_user_dir }}/venv/lib/python{{ ansible_test_python}}/site-packages/ara/plugins/callbacks"
-
-    - name: Enable persistent connection logging
-      ini_file:
-        path: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}/test/integration/{{ ansible_test_command }}.cfg"
-        section: persistent_connection
-        option: log_messages
-        value: true
-
-    - name: Set ansible log path
-      ini_file:
-        path: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible'].src_dir }}/test/integration/{{ ansible_test_command }}.cfg"
-        section: defaults
-        option: log_path
-        value: ~/ansible-debug.txt
```

Comment:
lets skip this change for now, I don't think that breaks anything in the role ATM.

Reply:
Why?

Reply:
I was going to push up the PR to move ARA into its own venv. Which is going on conflict on this. 

My concern as now, is we change the dependency order a little, also minimized the change

Reply:
I will be happy to do the Ara role, but I also want to see this one merged...